build.py

```diff
@@ -17,9 +17,13 @@
 
 """Main build script."""
 
+import gvsbuild.groups  # noqa: F401
+import gvsbuild.projects  # noqa: F401
+import gvsbuild.tools  # noqa: F401

```

Comment:
this looks wrong from a cursory view: the lines below try to adjust the path before importing more things, no?

Reply:
Hi @pbor, thanks for pointing this out. I am not sure why these imports got pulled up, but I moved them back down to where they were before.